src/model/item-data.c

```diff
@@ -249,15 +249,25 @@ void item_data_set_pos (ItemData *item_data, Coords *pos)
 void item_data_move (ItemData *item_data, const Coords *delta)
 {
 	ItemDataPriv *priv;
+	gboolean handler_connected;
 
 	g_return_if_fail (item_data != NULL);
 	g_return_if_fail (IS_ITEM_DATA (item_data));
 
 	if (delta == NULL)
 		return;
 
+	if (!delta->x && !delta->y)
```

Comment:
these are floats and thus this will always trigger, use `fabs(x) < EPS`

Reply:
Without it, it would perform unnecessary and very expensive matrix multiplication.

Reply:
Please read again what I wrote, the check is necessary, but it will never trigger as given above.

Reply:
I've just amended it !